MovieGPT-api/src/main/java/lt/techin/group/project/model/Media.java

```diff
@@ -0,0 +1,33 @@
+package lt.techin.group.project.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lt.techin.group.project.rest.dto.MediaDto;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Remove "Data" annotation, because "Entity" generate Database through JPA with given class states. With "Data" it gives unnecessary methods and cause problems in database replace with Getter, Setter.

Reply:
Solved, used Getter and Setter instead